### ISSUE TITLE
Bug in validation

### ISSUE THREAD

**Issue by QG-phy** [deepmodeling/DeePTB#53]
### Describe the bug

in train, the validation loss is sum of the loss from all the test data set, which should not be that. 

### Expected behavior

change to show only one test batch loss.
